runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -530,6 +560,18 @@ class JITServerAOTCache
    static AOTHeaderKey getRecordKey(const AOTCacheAOTHeaderRecord *record)
       { return { record->data().header() }; }
 
+   struct ThunkKey
```

Comment:
This struct is identical to `ClassLoaderKey`, which could be renamed to e.g. `StringKey` and used for both class loader record keys and thunk record keys, also avoiding code duplication for the hash and equals functions.